src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Square implements Figure {
+    private String color;
+    private double side;
+
+    public Square(String color, double side) {
```

Comment:
Creating a square, it's better not to pass expressions directly in the constructor. For better readability and maintainability, consider creating separate variables for `color` and `side` before passing them to the constructor.